hugolib/site_render.go

```diff
@@ -346,6 +347,7 @@ func (s *Site) renderSitemap() error {
 		if page.Sitemap.Filename == "" {
 			page.Sitemap.Filename = sitemapDefault.Filename
 		}
+		// Sitemap.Exclude isn't listed here because bools default to false, so it's done for us.
```

Comment:
If we're going to add `sitemapDefault.Exclude`, we should handle the scenario of `sitemapDefault.Exclude = true`.  However, I don't see how we make that work without `page.Sitemap.Exclude` being of type `*bool`.  If it's `nil`, we know the user didn't set the value.  If it's `false`, we know they mean to override the default.

A couple questions here:  Can we store front matter values into a pointer?  Do we need to add `cast.ToBoolP` to the cast library?

Or am I overthinking everything?  :wink:

Reply:
It's already set to a boolean.